client/my-sites/sidebar/style.scss

```diff
@@ -41,6 +41,7 @@ $brand-text: "SF Pro Text", $sans;
 		// Resetting these colors to prevent bleeding from dashboard color schemes
 		--color-sidebar-text: var(--studio-gray-80);
 		--color-sidebar-text-alternative: var(--studio-gray-40);
+		--color-sidebar-text-alternative-selected: var(--studio-gray-10);
```

Comment:
Instead of using grey `--studio-gray-10` here, what if we used an RGBA of --studio-white so that the text becomes light blue, instead of grey? I believe this will work. 

So instead of:

`--color-sidebar-text-alternative-selected: var(--studio-gray-10);`

We'd have something like:

`--color-sidebar-text-alternative-selected: rgba(var(--studio-white), 0.7);`

Reply:
Unsure why I cant get the variable working like that, but can simply use rgba(255, 255, 255, 0.7);